src/LoginForm/LoginForm.js

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from 'react'
+import './LoginForm.css'
+
+class LoginForm extends Component {
+  constructor() {
+    super();
+
+    this.state = {
+      name: '',
+      purpose: ''
+    }
+  }
+  updateChange = (e) => {
+    e.preventDefault();
+    this.setState({[e.target.name]: e.target.value})
```

Comment:
Nice way to make this dynamic for both the user's name as well as their purpose even though you are using buttons for purpose. sweeeet